python/py-gacode/Portfile

```diff
@@ -5,7 +5,11 @@ PortGroup           python 1.0
 PortGroup           compilers 1.0
 
 name                py-gacode
-version             0.40
+version             0.50
+set pypi_name       pygacode
+distname            ${pypi_name}-$version
+master_sites        pypi:[string index ${pypi_name} 0]/${pypi_name}
+
```

Comment:
instead of adding the three lines above, please just set python.rootname; the python PG will take care of the rest then. In other words, just do ```python.rootname pygacode```

Reply:
Good solution.